templates/devshells/haskell/dev-shell.ncl

```diff
@@ -1,9 +0,0 @@
-let builders = (import "./nickel.lock.ncl").nickel-nix.builders in
-
-{
-  ghcVersion = "927",
```

Comment:
Looks like not all templates are the same after all. Without this line we get this in CI:

```
building '/nix/store/dy3xs2il3qy9f6g92sipk5b9viy1ym4b-nickel-inputs.json.drv'...
nickel-inputs.json> error: missing definition for `ghcVersion`
nickel-inputs.json>     ┌─ /nix/store/3g0ymzm2z6dvls62y0s73g12rb8j63ip-5gm6jqx5lkzfxpslkjmc22b037g48b8i-source/builders.ncl:274:7
nickel-inputs.json>     │
nickel-inputs.json> 274 │       ghcVersion, # User-defined. To keep in sync with the one used by stack
nickel-inputs.json>     │       ^^^^^^^^^^ required here
nickel-inputs.json>     │
nickel-inputs.json>     ┌─ /nix/store/nm7k7izblms5h2rk62dswl420crlpp5g-h0vdahwbi2fbhaccnwik204z8vcmv7fz-source/dev-shell.ncl:3:1
nickel-inputs.json>     │
nickel-inputs.json>   3 │ ╭ {
nickel-inputs.json>   4 │ │   output = {
nickel-inputs.json>   5 │ │     name = "nickel-shell",
nickel-inputs.json>   6 │ │   }
nickel-inputs.json>   7 │ │ }
nickel-inputs.json>   8 │ │ & nixel.builders.HaskellStackShell
nickel-inputs.json>     │ ╰──────────────────────────────────' in this record
```